site/docs/v1/tech/apis/scim/_scim-group-request-body.adoc

```diff
@@ -0,0 +1,29 @@
+Because SCIM schemas are completely customizable, there are no required fields for the incoming request other than the common fields that all SCIM requests must include to match the specification.
+
+The following is an example of a typical SCIM Group create request body a SCIM client might send. However, your incoming request lambda could use these values in completely different ways to populate the FusionAuth User object if you choose.
+
+This is taken from the SCIM Schema RFC describing a SCIM User schema. For the full specification you can find the RFC link:https://datatracker.ietf.org/doc/html/rfc7643#section-4.2[here].
+
+==== Request Body
+[.api]
+include::/docs/v1/tech/apis/scim/_scim-common-body.adoc[]
+
+[field]#displayName# [type]#[String]#::
+The text that could be used to represent this Group in lists, etc.
+
+[field]#members# [type]#[Array]#::
+This list of members included in this Group.
+
+[field]#member.displayName# [type]#[String]#::
+
+[field]#meta.$ref# [type]#[String]#::
+The URI that can be used to retrieve data about the Group member.
```

Comment:
According to the spec the group member is any SCIM resource (User, EnterpriseUser, Group). In FusionAuth it would always be a User that is a member of the group for now at least because I don't believe we support sub groups.

That was a typo in the field though. It should be members[x].$ref. Fixing it.

Reply:
typo fixed.